.gitignore

```diff
@@ -6,4 +6,4 @@ test/dummy/db/*.sqlite3-journal
 test/dummy/log/*.log
 test/dummy/tmp/
 test/dummy/.sass-cache
-.DS_Store
```

Comment:
This is probably good to have in here.

Reply:
I don't even remember removing this...will add back

Reply:
![image](https://user-images.githubusercontent.com/13431788/135523002-ea59b05f-a7e8-46b4-9c64-eece22ba1524.png)
I'm confused

Reply:
Added back .DS_Store to gitignore